firstpr/pr_m.md

```diff
@@ -0,0 +1,6 @@
+#Prathap
+
+CS Student. Now a working professional for a big bank.
+
+I never got a chance to work on open source or UI dev. So, trying to improve my skills through this project.
```

Comment:
before the first full stop insert 'project'

Reply:
Thanks, file is updated